glmmTMB/src/Makevars

```diff
@@ -0,0 +1,7 @@
+PKG_LIBS = $(LAPACK_LIBS) $(BLAS_LIBS) $(FLIBS) $(SHLIB_OPENMP_CXXFLAGS)
+## FLIBS unneeded but harmless (?)
+# 1.2.4 Using C++11 code
+CXX_STD = CXX11
+
+# Tell Armadillo to use 64BIT_WORD, which removes the matrix size limit constraint
+PKG_CXXFLAGS=$(SHLIB_OPENMP_CXXFLAGS) -DARMA_64BIT_WORD
```

Comment:
Most of this is redundant. Can be reduced to:

```
PKG_LIBS = $(SHLIB_OPENMP_CXXFLAGS)
PKG_CXXFLAGS = $(SHLIB_OPENMP_CXXFLAGS)
```

Reply:
I thought I got this from an authoritative source, but now I can't relocate it. (Probably [here](https://www.r-bloggers.com/2017/08/setting-up-optional-openmp-support-with-rcpparmadillo/), which (a) is about using RcppArmadillo, which we don't; and (b) was probably what got me going down the `configure` rabbit hole in the first place. I'm all for simplifying if it works ...